these errors are also related to `multidict` - but this time they occur only from 6.0.1, but not with 5.2.0 or 6.0.0 😕 
```
Error: homeassistant/components/http/ban.py:120: error: No overload variant of "get" of "Mapping" matches argument type "str"  [call-overload]
homeassistant/components/http/ban.py:120: note: Possible overload variants:
homeassistant/components/http/ban.py:120: note:     def get(self, key: istr) -> Optional[str]
homeassistant/components/http/ban.py:120: note:     def [_T] get(self, istr, Union[str, _T]) -> Union[str, _T]
Error: homeassistant/helpers/config_entry_oauth2_flow.py:135: error: No overload variant of "get" of "Mapping" matches argument type "str"  [call-overload]
homeassistant/helpers/config_entry_oauth2_flow.py:135: note: Possible overload variants:
homeassistant/helpers/config_entry_oauth2_flow.py:135: note:     def get(self, key: istr) -> Optional[str]
homeassistant/helpers/config_entry_oauth2_flow.py:135: note:     def [_T] get(self, istr, Union[str, _T]) -> Union[str, _T]
Error: homeassistant/components/buienradar/camera.py:134: error: No overload variant of "get" of "Mapping" matches argument type "str"  [call-overload]
homeassistant/components/buienradar/camera.py:134: note: Possible overload variants:
homeassistant/components/buienradar/camera.py:134: note:     def get(self, key: istr) -> Optional[str]
homeassistant/components/buienradar/camera.py:134: note:     def [_T] get(self, istr, Union[str, _T]) -> Union[str, _T]
Found 3 errors in 3 files (checked 4578 source files)
```

--- a/homeassistant/package_constraints.txt
+++ b/homeassistant/package_constraints.txt
@@ -95,3 +95,7 @@ pytest_asyncio==1000000000.0.0
 # https://github.com/jkeljo/sisyphus-control/issues/6
 python-engineio>=3.13.1,<4.0
 python-socketio>=4.6.0,<5.0
+
+# Constrain multidict to avoid typing issues
+# https://github.com/home-assistant/core/pull/64792
+multidict<6.0.0

--- a/script/gen_requirements_all.py
+++ b/script/gen_requirements_all.py
@@ -120,6 +120,10 @@ pytest_asyncio==1000000000.0.0
 # https://github.com/jkeljo/sisyphus-control/issues/6
 python-engineio>=3.13.1,<4.0
 python-socketio>=4.6.0,<5.0
+
+# Constrain multidict to avoid typing issues
+# https://github.com/home-assistant/core/pull/64792
+multidict<6.0.0
 """
 
 IGNORE_PRE_COMMIT_HOOK_ID = (

